test/verify/check-machines

```diff
@@ -2770,6 +2770,9 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b = self.browser
         m = self.machine
 
+        m.execute("ip link add virbr2 type bridge")
```

Comment:
How sure are you that virbr2 won't collide with anything that libvirt could create by itself? Could it be something more specific, like "virbrtest" or so?

Reply:
I can do that - I thought virbr0 only is libvirt created but good point.